src/autonlp/metrics.py

```diff
@@ -26,14 +26,19 @@ def print(self):
         payload = {"project_id": self.project_id, "username": self.username}
         resp = http_post(path="/models/metrics", token=self._token, suppress_logs=True, payload=payload)
         json_jobs = json.loads(resp.json())
-        columns = json_jobs.keys()
-        print_logs = PrettyTable([col for col in columns])
-        print_logs.title = f"{BOLD_TAG}🚀 AutoNLP Model Metrics for project: {self.project_name}{RESET_TAG}"
-        indexes = json_jobs["model_id"].keys()
-        for idx in indexes:
-            line = [v[idx] if k == "model_id" else float(v[idx]) for k, v in json_jobs.items()]
-            print_logs.add_row(line)
+        if len(json_jobs) > 0:
+            columns = json_jobs.keys()
+            print_logs = PrettyTable([col for col in columns])
+            print_logs.title = f"{BOLD_TAG}🚀 AutoNLP Model Metrics for project: {self.project_name}{RESET_TAG}"
+            indexes = json_jobs["model_id"].keys()
+            for idx in indexes:
+                line = [v[idx] if k == "model_id" else float(v[idx]) for k, v in json_jobs.items()]
+                print_logs.add_row(line)
 
-        print_logs = print_logs.get_string(sortby="eval_loss")
-        print(print_logs)
-        print("")
+            print_logs = print_logs.get_string(sortby="eval_loss")
+            print(print_logs)
+            print("")
+            resp = http_get(path=f"/zeus/cost/{self.project_id}", token=self._token, suppress_logs=True).json()
+            print(f"Total cost incurred: {resp.get('cost_usd')}")
```

Comment:
Let's add the currency and keep two decimals:

```suggestion
            print(f"Total cost incurred: USD {resp.get('cost_usd'):.2f}")
```